plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -105,6 +105,11 @@ public function __construct() {
 				'description'     => __( 'Enable the high performance order storage feature.', 'woocommerce' ),
 				'is_experimental' => true,
 			),
+			'cart_checkout_blocks' => array(
```

Comment:
The arrows here are not correctly aligned resulting in a PHPCS error: https://github.com/woocommerce/woocommerce/actions/runs/3928725851/jobs/6716673300#step:6:20.

Reply:
Nice catch! 👍

It's weird! Running PHP lint locally shows me different results than the ones detected in CI.